src/Tribe/Date_Utils.php

```diff
@@ -246,14 +246,14 @@ public static function datetime_from_format( $format, $date ) {
 		/**
 		 * Returns the date only.
 		 *
-		 * @param int|string $date        The date (timestamp or string).
-		 * @param bool       $isTimestamp Is $date in timestamp format?
-		 * @param string|null $format The format used
+		 * @param int|string  $date         The date (timestamp or string).
+		 * @param bool        $is_timestamp Whether or not $date is in timestamp format.
+		 * @param string|null $format       The format used.
 		 *
 		 * @return string The date only in DB format.
 		 */
-		public static function date_only( $date, $isTimestamp = false, $format = null ) {
-			$date = $isTimestamp ? $date : strtotime( $date );
+		public static function date_only( $date, $is_timestamp = false, $format = null ) {
+			$date = $is_timestamp ? $date : strtotime( $date ?? 'now' );
```

Comment:
I'd just add a note to the docblock that $date defaults to 'now' if it's empty

Reply:
Updated.